spring-cloud-task-stream/pom.xml

```diff
@@ -42,7 +42,13 @@
 		</dependency>
 		<dependency>
 			<groupId>org.springframework.cloud</groupId>
-			<artifactId>spring-cloud-stream-test-support</artifactId>
+			<artifactId>spring-cloud-stream-test-support-internal</artifactId>
+			<version>1.0.0.BUILD-SNAPSHOT</version>
+			<scope>test</scope>
+		</dependency>
+		<dependency>
+			<groupId>org.springframework.cloud</groupId>
```

Comment:
@cppwfs yeah, sorry it was my mistake to keep the integration test inside along with the unit tests.
With the recent changes to this PR, I have moved the integration test into a separate sub-project (which uses redis binder) and let the unit tests run with the `TestSupportBinder`. This way, the integration test will skip running the test if Redis not available (instead of failing).